reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,161 @@
+information about creating and using pipeline expressions.
+
+### Clone Server Group
+Deploys a new Server Group which is a copy of the specified Server Group.
+
+### Deploy
+Deploys the previously baked or found image using the specified deployment
+strategy. Spinnaker provides built-in support for both red/black (also known as
+blue/green) and highlander deployment strategies. You can also choose to deploy
+with no impact on existing Server Groups, or build your own custom deployment
+strategy.
+
+### Destroy Server Group
+Deletes a Server Group and its resources from the specified Cluster. You must
+specify whether you want to delete the newest, oldest, or previous
+(second-most-recently deployed) Server Group when this stage starts.
+
+### Disable Cluster
+Disables the specified Cluster, which means that the cluster remains up but
+stops handling traffic. If desired, you can leave a specific number of Server
+Groups running while the rest of the cluster is disabled.
+
+### Disable Server Group
+Disables a Server Group, which means that the Server Group remains up but stops
+handling any traffic. This makes it easy to both route traffic to a new
+Server Group and roll back those changes if necessary. You must choose whether
+to disable the Server Group which is newest, oldest, or previous
+(second-most-recently deployed) when this stage starts.
+
+### Enable Server Group
+Enables a Server Group, which means that the Server Group will begin handling
+traffic again.
+
+### Find Artifact From Execution
+Find and bind an artifact from another pipeline execution.
+
+### Find Image From Cluster
+Finds an image to deploy from an existing Cluster. You must specify the Cluster,
+Server Group, and image name such that there is exactly one match in order for
+this stage to succeed.
+
+### Find Image From Tags
+Finds an image to deploy from tags.
+
+### Jenkins
+Runs a Jenkins job. You must [set up Jenkins](/setup/ci/jenkins/) in order to
+use this stage.
+
+### Manual Judgment
+Waits for user approval before continuing.
+
+### Pipeline
+Runs an existing pipeline: you can select any pipeline from this application and
+run it as a sub-pipeline.
+
+
+### Resize Server Group
+Resize the oldest, newest, or previous (second-most-recently deployed) Server
+Group. You can resize the Server Group by either a percentage of its current
+size or a specific amount. The resizing strategies available to you are:
+* **Scale Up**, which increases the size of the target Server Group.
+* **Scale Down**, which decreases the size of the target Server Group.
+* **Scale to Cluster Size**, which increases the size of the target Server Group
+to match the largest Server Group in the Cluster. Optionally, you can specify
+additional capacity to add as well.
+* **Scale to Exact Size**, which adjusts the size of the target Server Group to
+match the specified capacity.
+
+### Rollback Cluster
+Roll back one or more regions in a Cluster.
+
+### Run Job
+Runs a container.
```

Comment:
Some of these first-word verbs are indicative and some are imperative. "Runs," here, versus "Roll back" in the previous one and "Scale" in the next one. Kind of a nit, but it might be cleaner to be consistent throughout.